src/gameboard.js

```diff
@@ -0,0 +1,80 @@
+const Gameboard = ((red, black) => {
+  const board = [];
+  const emptySquare = { hasMarker: false };
+
+  const initialize = () => {
+    for (let i = 0; i < 8; i++) {
+      const row = [];
+      for (let j = 0; j < 8; j++) {
+        const square = emptySquare;
+        row.push(square);
+
+        if (((i + j) % 2 === 1) && (i < 3)) {
+          square.hasMarker = red;
+        }
+
+        if (((i + j) % 2 === 1) && (i > 4)) {
+          square.hasMarker = black;
+        }
+      }
+      board.push(row);
+    }
+
+    console.log('Gameboard initialized');
+  };
+
+  initialize();
+
+  const getSquare = (index) => {
```

Comment:
this function is unnecessary.  i'm not sure what `index` as an input would be, other than an array `[x, y]`, and in that case we can just write the call `board[x][y]` instead of `getSquare([x, y])`

Reply:
If board is a private variable inside gameboard, board[x][y] would return undefined.